colin-api/src/colin_api/resources/event.py

```diff
@@ -60,25 +60,27 @@ def get(corp_type, event_id):
 
 
 @cors_preflight('GET')
-@API.route('/event/corp_num/<string:corp_num>')
+@API.route('/event/corp_num/<string:corp_num>/<string:event_id>')
```

Comment:
so you're sure adding the event_id here to the route won't break anything?

Reply:
i did not find any reference to this endpoint